README.md

```diff
@@ -0,0 +1,9 @@
+# A ROS2 RMW implementation for Vortex OpenSplice
```

Comment:
Please add a space between "ROS" and "2".

Same below.

Reply:
ok done